package.json

```diff
@@ -74,4 +77,4 @@
   "engines": {
     "node": ">=18.18.0"
   }
-}
\ No newline at end of file
+}
```

Comment:
My editor keeps adding this change. I think there should be an extra line here given that there is one at the end of other files in the project 🤔